kcqrs-core/src/main/kotlin/com/clouway/kcqrs/core/JsonSerializer.kt

```diff
@@ -0,0 +1,16 @@
+package com.clouway.kcqrs.core
+
+import com.google.gson.JsonIOException
+import com.google.gson.JsonSyntaxException
+import java.lang.reflect.Type
+
+/**
+ * @author Vasil Mitov <vasil.mitov@clouway.com>
+ */
+interface JsonSerializer {
```

Comment:
MessageFormat is already added for these purposes.

Reply:
Will be removed.